site/concepts/packaging/README.md

```diff
@@ -68,7 +68,7 @@ Here are the core principles around `subpackages` concept
    `--recurse-subpackages(-R)` is provided. (only available with [cfg] commands currently
    and the default value of `-R` flag might vary for each command)
 
-[source functions]: ../functions/#source-function
+[source functions]: /concepts/functions/#source-function
 [update]: https://googlecontainertools.github.io/kpt/guides/consumer/update/
```

Comment:
As a follow up PR: There shouldn't be fully qualified links to other kpt packages (This will also mask broken links)

Reply:
Noted: should have caught some in more recent commits but good to keep an eye out.